tables/output_tables.Rmd

```diff
@@ -143,19 +143,20 @@ soi_df <- tmb_hist %>%
     ),
     # add Lynch syndrome
     cancer_predispositions = case_when(
-      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "NF-1, Lynch syndrome",
+      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "Lynch syndrome",
+      sample_id == "7316-515" ~ "Other inherited conditions NOS",
```

Comment:
I'm not sure I see where `"Other inherited conditions NOS"` comes from. I see no genetic evidence for NF-1.

Reply:
NF1 and other were the initial values in the cancer predisposition column but I removed NF1 here. Actually it could be that NOS was entered because of NF1 in the path report but we probably can't ever be sure.